models/linkages/R/write.restart.LINKAGES.R

```diff
@@ -77,10 +70,18 @@ write.restart.LINKAGES <- function(out.dir, runid, time, settings, analysis.vec,
   }
   
   spp.params <- spp.params.default[spp.params.save,]
+  
+  biomass_function<-function(dbh){ #kg/tree
+    .1193 * dbh^2.393 + ((slta+sltb*dbh)/2)^2 * 3.14 * fwt * frt * .001
+  }
+  merit<-function(dbh){
+    (b_obs - biomass_function(dbh))^2
```

Comment:
there are non-local variables in both of these functions : `slta`,`sltb`,`fwt`,`frt`,`b_obs`

if you pass them as an argument explicitly, it would be less error-prone